src/network.cpp

```diff
@@ -162,15 +183,35 @@ String templateProcessor(const String &var) {
       opts.concat("</option>");
     }
     return opts;
+  } else if (var == "AVAIL_NETWORK_OPTIONS") {
+    String wifi_network_opts;
+    wifi_network_opts.reserve(65500);
```

Comment:
This is a huge overkill, can't believe it didn't blow up in your face when you were testing it lol. 

Each option is ~35 bytes of html + AP name. Make this 2kB, this should cover most cases, especially now that most WiFi networks are moving to 5Ghz band making them invisible to Owie.

I'd also like to retain the textbox to still allow joining to invisible WiFi networks. Can you please instead just render the list of networks available and maybe just copy the name into the text field when one of the scan result items gets clicked on?

Reply:
I will reduce the predefined string length.

Regarding the list of networks.. If you look at the HTML file, you will see that it still remains an standard input. The dropdown is only a additional datalist.
You'r able to either select from the list (if the field is empty) or type your own one.